From 8331f69679a14463a3abf14d164dbe304bb5dbe2 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Thu, 2 Feb 2023 13:05:57 +0800
Subject: [PATCH] Revert "Linux 5.10.166"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 8d823aaa220eebec88c9f307225d3e163252ea95.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index efdfb40a82ab..5fbff8603f44 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 166
+SUBLEVEL = 165
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

